frontends/election-manager/src/components/print_all_ballots_button.test.tsx

```diff
@@ -0,0 +1,248 @@
+import fetchMock from 'fetch-mock';
+import { act, render, screen, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { electionMinimalExhaustiveSampleDefinition } from '@votingworks/fixtures';
+import { fakeLogger, LogEventId } from '@votingworks/logging';
+import { MemoryCard, MemoryHardware, typedAs } from '@votingworks/utils';
+import React from 'react';
+import { fakeKiosk } from '@votingworks/test-utils';
+import { hasTextAcrossElements } from '../../test/util/has_text_across_elements';
+import { renderInAppContext } from '../../test/render_in_app_context';
+import {
+  authenticateWithAdminCard as authenticateWithAdminCardNoDefinition,
+  authenticateWithSuperAdminCard,
+} from '../../test/util/authenticate';
+import { createMemoryStorageWith } from '../../test/util/create_memory_storage_with';
+import { App } from '../app';
+import {
+  PrintAllBallotsButton,
+  PRINTER_WARMUP_TIME,
+  TWO_SIDED_PRINT_TIME,
+} from './print_all_ballots_button';
+import { MachineConfig } from '../config/types';
+import { fakePrinter } from '../../test/helpers/fake_printer';
+
+jest.mock('../components/hand_marked_paper_ballot');
+
+fetchMock.get(
+  '/machine-config',
+  typedAs<MachineConfig>({
+    machineId: '0000',
+    codeVersion: 'TEST',
+  })
+);
+
+beforeEach(() => {
+  jest.useFakeTimers();
+});
+
+async function authenticateWithAdminCard(card: MemoryCard) {
+  return authenticateWithAdminCardNoDefinition(
+    card,
+    electionMinimalExhaustiveSampleDefinition
+  );
+}
```

Comment:
Same comment here as earlier about the `authenticateWithAdminCardNoDefinition` function name being a bit confusing (though the fix here may look different than the fix in `app.test.tsx` since we're using a different election definition)